docs/GettingStarted.md

```diff
@@ -36,144 +38,114 @@ This will start a container. Check they are running
 
 ## Creating networked endpoints
 
-Now you can start any other containers that you want within the cluster, using normal docker commands. To get Calico to network them, simply add `-e CALICO_IP=<IP address>` to specify the IP address that you want that container to have.
+The experimantal channel version of Docker introduces a new flag to `docker run` to network containers:  `--publish-service <service>.<network>.<driver>`.
 
-(By default containers need to be assigned IPs in the `192.168.0.0/16` range. Use `calicoctl` commands to set up different ranges if desired)
+ * `<service>` is the name by which you want the container to be known on the network.
+ * `<network>` is the name of the network to join.  Containers on different networks cannot communicate.
+ * `<driver>` is the name of the network driver to use.  Calico's driver is called `calico`.
 
 So let's go ahead and start a few of containers on each host.
 
-On core-01
-
-    docker run -e CALICO_IP=192.168.1.1 --name workload-A -tid busybox
-    docker run -e CALICO_IP=192.168.1.2 --name workload-B -tid busybox
-    docker run -e CALICO_IP=192.168.1.3 --name workload-C -tid busybox
+On ubuntu-0
 
-On core-02
+    docker run --publish-service srvA.net1.calico --name workload-A -tid busybox
+    docker run --publish-service srvB.net2.calico --name workload-B -tid busybox
+    docker run --publish-service srvC.net1.calico --name workload-C -tid busybox
 
-    docker run -e CALICO_IP=192.168.1.4 --name workload-D -tid busybox
-    docker run -e CALICO_IP=192.168.1.5 --name workload-E -tid busybox
+On ubuntu-1
 
-At this point, the containers have not been added to any policy profiles so they won't be able to communicate with any other containers.
+    docker run --publish-service srvD.net3.calico --name workload-D -tid busybox
+    docker run --publish-service srvE.net1.calico --name workload-E -tid busybox
 
-Create some profiles (this can be done on either host)
+By default, networks are configured so that their members can communicate with one another, but workloads in other networks cannot reach them.  A, C and E are all in the same network so should be able to ping each other.  B and D are in their own networks so shouldn't be able to ping anyone else.
 
-    ./calicoctl profile add PROF_A_C_E
-    ./calicoctl profile add PROF_B
-    ./calicoctl profile add PROF_D
+You can find out a container's IP by running
 
-When each container is added to calico, an "endpoint" is registered for each container's interface. Containers are only allowed to communicate with one another when both of their endpoints are assigned the same profile. To assign a profile to an endpoint, we will first get the endpoint's ID with `calicoctl container <CONTAINER> endpoint-id show`, then paste it into the `calicoctl endpoint <ENDPOINT_ID> profile append [<PROFILES>]`  command.
+    docker inspect --format "{{ .NetworkSettings.IPAddress }}" <container name>
```

Comment:
There is a not saying that libnetwork currently has issues in multi-host configurations when it comes to pinging.  I confirmed this by not being able to ping from Container-A on Host-1 to Container-E on Host-2.  I think we have to keep it like this for now.